config/wrapper/env.conf

```diff
@@ -1,7 +1,14 @@
-[repo]
-avocado = https://github.com/avocado-framework/avocado.git
-avocado_vt = https://github.com/avocado-framework/avocado-vt.git
-tests = https://github.com/avocado-framework-tests/avocado-misc-tests.git
+[framework]
+# Usage examples:
+# (avocado-framework, '')  -- Installs latest released version
+# ('avocado-framework', 80.0)  -- Installs specificed version, make sure you change other entries aswell
+# ('avocado-framework', 'git+https://github.com/avocado-framework/avocado.git') -- Install version from the given git repo
+base = [('avocado-framework', ''), ('avocado-framework-plugin-varianter-yaml-to-mux', '')]
```

Comment:
One question here: Do we need an option for version of optional plugins here?  I understand the need for a kvm (avocado-vt) version. Considering optional plugins are maintained as part of avocado-framework itself we can use the same versioning for additional/optional plugins also by default?

Reply:
you are right, but there are few scenarios which might demand the version tag,

1) consider we want to use a different avocado version and we want to match the version to that modified one, otherwise it will install the latest which might leave env unstable.

2) same case with above but with custom git with different version as latest. and even that case it will be handy to choose the plugin version matching.

after all, it the same processing, so it good to have it enabled and provide the flexibility to user.

Reply:
Agreed, it does provide the flexibility for us. Just concerned that we need to repeat on all places, but we can leave it to the user.